DESCRIPTION

```diff
@@ -551,7 +551,8 @@ Imports:
     rlang,
     stringr,
     tibble (>= 3.0.0),
-    tidyr (>= 1.0.0)
+    tidyr (>= 1.0.0),
+    vctrs
```

Comment:
this dependency can be removed again when we register the methods unconditionally